packages/list-react/src/List.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { ReactNode } from "react";
+
+interface Props {
+    children: ReactNode;
+    className?: string;
+    ordered?: boolean;
+}
+
+export const List = ({ children, className = "jkl-p", ordered }: Props) => {
+    const componentClassName = "jkl-list".concat(
+        className ? ` ${className}` : "",
+        ordered ? " jkl-list--ordered" : " jkl-list--bullet",
+    );
+
+    const C = ordered ? "ol" : "ul";
+
+    return <C className={componentClassName}>{children}</C>;
+};
```

Comment:
Et forslag (som @lfbergee sikkert blir glad for 😅) er å gjøre dette til en factory og eksportere to komponenter, `OrderedList` og `UnorderedList`. Funker flott som det er nå, men med to komponenter speiler vi både funksjonaliteten og "native" måte å gjøre det på i enda større grad. Du får vurdere!